packages/suite-desktop-ui/src/support/DesktopUpdater/Downloading.tsx

```diff
@@ -56,42 +46,31 @@ export const Downloading = ({ hideWindow, progress }: DownloadingProps) => {
     }, [step]);
 
     return (
-        <Modal
-            headerComponent={
```

Comment:
This was here because closing the modal during update could lead to confusion - does downloading continue or not? See https://github.com/trezor/trezor-suite/pull/7815#discussion_r1138806326.